Apps/frontend/src/labels.js

```diff
@@ -8,6 +8,13 @@ export const LABEL_BUTTON_LOAD_CHANNEL_CONFIG = "Load";
 export const LABEL_BUTTON_UPDATE_CHANNEL_CONFIG = "Save";
 export const LABEL_BUTTON_CANCEL_CHANNEL_CONFIG = "Cancel";
 
+export const TOOLTIP_BUTTON_GND = "GND";
```

Comment:
Not your piece of the cake (and I sadly currently don't know who introduced it), but we already have a `cponst` file, why introduce a new labels one?

Reply:
I don't know why/who seperated this but I actaully like it because it seperated text-related and functionallity-related constants

Reply:
Find it okay to create an extra file. 
On the other hand, it could also be seen as part of a svelte component. If I wanted to change something without knowing anything about the project, I would probably look into the components first and then go back to the constants.

Reply:
I'm not used to having an extra file only for labels (usually directly in const file), but think it's fine as it is :)